### PR TITLE
fix: Markdownlint cleanup for JA

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Review by mfuji09 — CHANGES_REQUESTED** [review]
Please correct a problem.

File: files/ja/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 1]
```diff
@@ -155,23 +155,21 @@ CDN は地理的に分散したサーバーネットワークで、ユーザー
 
 ## ページ構造の例
 
-· `{{htmlelement('html')}}`
-
-- · `{{htmlelement('head')}}`
```
Comment (nschonni) [inline files/ja/learn/html/howto/author_fast-loading_html_pages/index.md:160]: - `{{htmlelement('html')}}`

File: files/ja/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 2]
```diff
@@ -155,23 +155,21 @@ CDN は地理的に分散したサーバーネットワークで、ユーザー
...
+- `{{htmlelement('head')}}`
```
[the remark this answers is not in the document]
Comment (nschonni) [reply files/ja/learn/html/howto/author_fast-loading_html_pages/index.md:159]: Thanks, I must have read it as 'head' twice, rather than unique calls

**Review by mfuji09 — APPROVED** [review]
LGTM.

**Pull request merged**